### DISCUSSION TITLE
Serverpod ORM only

### DISCUSSION THREAD

**Discussion by chimon2000** [serverpod/serverpod#1325] [Q&A] [answered]
Is it possible to use Serverpod's ORM without the rest of the framework?

**Comment by odrya12** [comment]
look extremely challenging, I don't recommend this.

**Comment by vlidholt** [comment, marked as answer]
Currently, this is (probably) not possible. May be sweet to make this an option in the future, but cannot make any promises if it will happen or any timeline.
